libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/Fields.hs

```diff
@@ -254,7 +254,7 @@ initialTxBody (Alonzo _) =
     initVI
     SNothing
     Set.empty
-    initValue
+    mempty
```

Comment:
There is something not quite right about using mempty here. All the other types use the initXXX, so why change it to mempty for Value?  There is a sort of self documentation that is being broken when we do this.

Reply:
Indeed, just laziness on my side.  Will change